src/main/java/co/dapi/DapiApp.java

```diff
@@ -297,6 +299,40 @@ public GetAccountsMetadataResponse getAccountsMetadata(String accessToken, Strin
         return this.m.getAccountsMetadata(accessToken, userSecret, operationID, userInputs);
     }
 
+
+    /**
+     * createPull talks to the CreatePull endpoint of Dapi, with this {@link DapiApp}'s appSecret,
+     * to continue a previous operation that required to provide some userInputs.
+     *
+     * @param transfer    the transfer details that we want to initiate.
+     * @param accessToken retrieved from the ExchangeToken process.
+     * @param userSecret  retrieved from the user login.
+     * @param operationID retrieved from the previous call's response.
+     * @param userInputs  built from the previous call's response, and the required user input.
+     * @return an {@link CreatePullResponse}.
+     * @throws IOException in case of trouble happened while executing the request or reading the response.
+     */
+    public CreatePullResponse createPull(ACH.CreatePull transfer, String accessToken, String userSecret, String operationID, UserInput[] userInputs) throws IOException {
+        return this.ach.createPull(transfer, accessToken, userSecret, operationID, userInputs);
+    }
+
+
+    /**
+     * getPull talks to the GetPull endpoint of Dapi, with this {@link DapiApp}'s appSecret,
+     * to continue a previous operation that required to provide some userInputs.
+     *
+     * @param accessToken retrieved from the ExchangeToken process.
+     * @param userSecret  retrieved from the user login.
+     * @param operationID OperationID of the createPull request
+     * @param userInputs  built from the previous call's response, and the required user input.
+     * @return an {@link GetPullResponse}.
+     * @throws IOException in case of trouble happened while executing the request or reading the response.
+     */
+    public GetPullResponse getPull(String accessToken, String userSecret, String operationID, UserInput[] userInputs) throws IOException {
```

Comment:
Rename to `getACHPull`.

Reply:
https://github.com/dapi-co/dapi-java/pull/18/commits/b06d400dd7b832a87c000dc14cc3cc5990b1da32